### ISSUE TITLE
Need: Email address of dreamer

### ISSUE THREAD

**Issue by SaskiaFuzzy** [kiezburn/dreams#26]
Hei everyone,

something is not quite correct in the display of information and / or in the question.

a) I cannot see the phone number of the dreamer in the dream. Usually, as an admin, I could see the email address and the phone number of the dreamer. With the current system I can not access the email-adress (which is about the most important thing!). I can also not access the phone number unless I use admin power and go to 'edit the dream' to check what they entered in the box.
FYI: IF I click on the 'dream crew' icon I should be taken to a user profile. The URL is f.e. https://dreams.kiezburn.org/users/701  and that could give me the email address, but currently it leass to an empty page 

In short: Need to access phone number & emailaddress (Priority here!) of dreamers to set up contact sheet.

**Comment by SaskiaFuzzy** [comment]
If the emailaddress cannot be accessed anymore because it is not displayed through the user profile we need to make another box in the sign up form asking people for their 'preferred email address for dream communication (don't worry, won't be public)' just like the phone number question and display that to admin right users.
